app/controllers/api/v0/search_controller.rb

```diff
@@ -64,7 +64,7 @@ class Api::V0::SearchController < Api::V0::BaseController
   def search
     started_at = Time.now
 
-    search_strategy_instance = Search::BookVersions::SearchStrategies::Factory.build(
+    search_strategy_instance = Books::SearchStrategies::S1::Factory.build(
```

Comment:
The factory was responsible for building the right instance of the search strategy, so I don’t think it can be inside the `S1` namespace?  (because it can create an `S2` strategy if that is what is passed in in the `search_strategy` argument)